packages/ui/src/memberships/components/MemberListItem/Fileds.tsx

```diff
@@ -12,9 +12,18 @@ export const colLayoutByType = (kind: MemberItemKind) => {
   const count = kind === 'Member' ? 20 : 76
   const total = 100
 
-  return kind === 'Member'
-    ? `${id}px ${name}px ${concil}px ${roles}px ${count}px ${count}px ${total}px ${total}px`
-    : `${name}px ${roles}px ${count}px ${count}px 96px 76px 54px`
+  switch (kind) {
+    case 'Member':
+      return `${id}px ${name}px ${concil}px ${roles}px ${count}px ${count}px ${total}px ${total}px`
+    case 'MyMember':
+      return `${name}px ${roles}px ${count}px ${count}px 96px 76px 54px`
+    case 'MyMemberships':
+      return '194px 200px 76px 76px 96px 76px 54px'
+    case 'Accounts':
+      return '276px repeat(4, 128px) 136px'
```

Comment:
Because we treat Membership as a different domain to Accounts those two shouldn't be mixed.

The way it was before defined was OK (as a styled-component override).